ambry-network/src/main/java/com.github.ambry.network/Transmission.java

```diff
@@ -132,8 +132,7 @@ public SocketAddress getRemoteSocketAddress() {
   /**
    * Close the connection for the socket channel
    */
-  public abstract void close()
-      throws IOException;
```

Comment:
not sure if this is a good change. Just because the current implementations do not throw `IOException`, does it mean that it is no longer relevant to the method? You might be able to make the same argument for more than one function in this class.